### DISCUSSION TITLE
Could we include a README file in our NuGet packages?

### DISCUSSION THREAD

**Discussion by stijnmoreels** [arcus-azure/arcus#207] [Ideas]
Starting from .NET 5, one can include a README file in the NuGet package generation. This README file is then shown on nuget.org as a first impression of the library. Maybe it's already enough to include the README of the GitHub repo?

**Comment by pim-simons** [comment]
Sounds like something we should include, the more docs the better 😉 
The README of the GitHub repo seems fine to me 👍🏻

**Reply by stijnmoreels** [reply]
Done in https://github.com/arcus-azure/arcus/issues/209
